packages/ui/dialogs/settlement/index.tsx

```diff
@@ -80,18 +80,16 @@ interface Props {
 
 interface State {
   amount?: string
-  formInputError?: string
   balance: number
   direction: string
-  txCost: string
-  cryptoCost: string
-  cryptoBalance: string
   pic?: string
   settlementType?: string
   friend: Friend
   fromPayPalRequest?: boolean
   pickerSelection: any
+  settlementInfo: any
   showPicker: boolean
+  transactionCosts: any
```

Comment:
The `any` type should be avoided wherever possible, in favor of a proper type specification. Usage of `any` negates the benefit of using TypeScript (i.e: type checking)! Is there a way to express the expected type here?

Reply:
I don't think there is a type for "object", I've only seen `any` used for this case.

Reply:
Is there any way to describe more fully the shape of that object? Or is it truly a free-for-all?

Reply:
Great question and I'm all for type-casting. I haven't seen anything in used to shape objects in Lndr, and it's rampant, but I'll look around at options.

Reply:
I defined both of those objects as interfaces and "shaped" their properties and updated the code.

The result generated 2 errors that were otherwise missed, so a great call!